### ISSUE TITLE
Compile TeX-file using the asyncronous capability of vim 8

### ISSUE THREAD

**Issue by cristobaltapia** [gerw/vim-latex-suite#31]
I was thinking that it would be nice if the plugin would compile the tex-file asyncronously. It should be pretty easy using the AsyncRun plugin [1]. 

[1] https://github.com/skywind3000/asyncrun.vim

**Comment by gerw** [comment]
Yes, this could be a nice idea. However, I use `latexmk` for continuous compilation (in another terminal window). Hence, I will not implement this.

**Comment by cristobaltapia** [comment]
I use it, too :P . I just was not sure if it was better to have it incorporated or not.
